README.md

```diff
@@ -39,6 +39,21 @@ the Kubernetes world:
 [5]: https://github.com/openshift/machine-config-operator/tree/master/docs
 [6]: https://github.com/UKHomeOffice/seccomp-config
 
+
+## Features
+
+The SPO's features are implemented for each one of the underlying
+supported technologies, namely: Seccomp, SELinux and AppArmor. 
+Here's the main features' status:
+
+|                                  | Seccomp | SELinux | AppArmor |
+|----------------------------------|---------|---------|----------|
+|             Custom Resource Type |   Yes   |   Yes   |    Yes   |
+|       Deploy profiles into nodes |   Yes   |   Yes   |    WIP   |
+| Remove profiles no longer in use |   Yes   |   Yes   |    WIP   |
+|          Profile Auto-generation |   Yes   |   Yes   |  Not yet |
+|             Audit log enrichment |   Yes   |    ?    |    Yes   |
```

Comment:
I am not sure whether log enrichment is working for SELinux, so kept it as `?`. Has anyone played with it?

Reply:
I would mark it as WIP now. I'm working on more tests this week and probably some enhancements to selinuxd that would make the policies easier. Plus, we're thinking about changing the policy format a bit to make it more kube-like. So while it works, I think saying WIP is more precise because it makes it clear to the user that there might be changes.

Reply:
btw what is profile auto-generation? How does it differ from log enrichement, is it using e.g. the BPF recorder? Because for SELinux, the audit logs are currently (and probably will stay that way) the only way to generate policies.

Reply:
Nice one @jhrozek. I changed it to WIP.

By `Profile auto-generation` I was trying to arrive at an agnostic way to covey the ability of autonomously generate profiles. For the time being I broke it down the feature and the approach used (ebpf/log).